src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(movie);
+            transaction.commit();
```

Comment:
It is recommended to use try-with-resources for read operations to ensure that the session is closed properly. However, since you are using a try-finally block here to handle transactions, it is acceptable.